src/utils/versions/getVersionsColors.ts

```diff
@@ -9,6 +9,7 @@ export const hashCode = (s: string) => {
     }, 0);
 };
 
+// TODO: colors used in charts as well, need to move to constants
```

Comment:
DIdn't move colors constants, so internal version imports won't be changed while it's being moved to open-source